### ISSUE TITLE
Feature Request: Watermarked Documents

### ISSUE THREAD

**Issue by imetandy** [BookStackApp/BookStack#1187]
**Describe the feature you'd like**
An option to easily add a watermark to documents, and choose its positioning (top left, top right, etc). 

**Describe the benefits this feature would bring to BookStack users**
This would be beneficial for using BookStack for policies in a work environment. For documents only viewed on BookStack, it's not really required, but say I wanted to print off a policy for another member of staff or third party company, I'd like to have company info (logo, web address) on the PDF. 

**Additional context**
I work as a single IT administrator for a company and use BookStack for Policies and documenting servers and applications we use here. 

Thanks for all your work so far on this project, the recent addition of shelves has been really beneficial, and has really complimented my slight OCD! 👍

**Comment by R0GGER** [comment]
This feature would be great! 
+1

**Comment by gdrnclaeys** [comment]
Agree, this would be a great feature to let people know that documentation is still in a draft stage for example.

**Comment by IhanSadiq** [comment]
This would be a great addition to add to the list of functionalities!

**Comment by senshiKimura** [comment]
Hi team of BookStack.
You do great job with BookStack and it's the best free knowledge tool easy to use in the world ^^
Could you add the WaterMark feature for able bookstack to become THE knowledge to use pls x)

**Comment by realviewch** [comment]
+1 ... Would be very nice!

**Comment by athornfam2** [comment]
+1 would like this feature as well.

**Comment by PlasmaDev5** [comment]
+1 this would be important for me

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/1187
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
